poster/poster.tex

```diff
@@ -0,0 +1,367 @@
+\end{block} %introduction block
+            \vskip1ex
+\begin{alertblock}{Data Collection}
+\begin{itemize}
+\item Took pictures of bananas and non-banana objects.
+\item Lighting, camera (Canon S90) and background were controlled. 
+\item Used 12 unique bananas at various stages to represent the three stages ripeness: unripe, ripe and overripe. 
+\item Non-banana objects: green pepper, apple, tomato, lemon and lime. 
+\item Pictures were resized and cropped to a square. 
+\item Incorporated each picture at $0\,^{\circ}$, $90\,^{\circ}$, $180\,^{\circ}$ and  $270\,^{\circ}$ of rotation to increase the number of pictures in the data set by four fold.
+\item Total of 928 images generated. 
+\end{itemize}
+\end{alertblock}
+ \vskip2ex
+ \begin{figure}
+ %%%%%% unripe %%%%%%%%%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{1_1.jpg}
+\end{subfigure}%
+ \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{1_2.jpg}
+\end{subfigure}%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{1_3.jpg}
+\end{subfigure}%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{1_4.jpg}
+\end{subfigure}\, %%%%%% ripe %%%%%%%%%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{2_1.jpg}
+\end{subfigure}%
+ \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{2_2.jpg}
+\end{subfigure}%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{2_3.jpg}
+\end{subfigure}%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{2_4.jpg}
+\end{subfigure}%
+\vskip .2in
+ %%%%%% overripe %%%%%%%%%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{0_1.jpg}
+\end{subfigure}%
+ \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{0_2.jpg}
+\end{subfigure}%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{0_3.jpg}
+\end{subfigure}%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{0_4.jpg}
+\end{subfigure}\,
+ %%%%%% non banana %%%%%%%%%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{3_1.jpg}
+\end{subfigure}%
+ \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{3_5.jpg}
+\end{subfigure}%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{3_17.jpg}
+\end{subfigure}%
+  \begin{subfigure}{.123\textwidth}
+  \centering
+\includegraphics[width=\textwidth]{3_13.jpg}
+\end{subfigure}%
+\vskip 1.ex
+\caption{Upper left: unripe bananas. Upper right: ripe bananas. Lower left: overripe bananas. Lower Right: non-banana objects.}
+\end{figure}
+
+\begin{block}{Methodology}
+ \begin{itemize}
+ \item Used various open source tools for features extraction and classification. 
```

Comment:
Should probably have a one sentence high level description of what we did, e.g:
"Used a pre-trained CNN to extract features from image set and classified those using SVM"
I know it's in the workflow but I'd like to see it written down.